src/webchat-ui/components/presentational/Header.tsx

```diff
@@ -133,8 +135,7 @@ const Header: FC<HeaderProps> = props => {
 
 	const isCompact =
 		!mainContentRef?.current ||
-		mainContentRef.current.scrollHeight > mainContentRef.current.clientHeight ||
-		!logoUrl;
+		mainContentRef.current.scrollHeight > mainContentRef.current.clientHeight;
```

Comment:
```suggestion
		mainContentRef.current.scrollHeight > mainContentRef.current.clientHeight || 
		!isChatOptionsButtonVisible;
```

Reply:
Since you removed the `!logoUrl `check, we need to add additional condition to make the header always compact in rating and chat options screens.

Reply:
I added your change. I hope we can clear up this mess with the internal navigation soon